### DISCUSSION TITLE
Suggestion: Search Bar

### DISCUSSION THREAD

**Discussion by Morndenkainen** [tsunamayo/Starship-EVO#5235] [Suggestions]
Played the game for an hour or so and one thing I noticed was that I was constantly swapping between blocks and having to search for them. I know this game isn't space engineers, but the search feature they have in the blocks menu when swapping blocks between the menu and the hotbar really helps with finding things quickly.. For example, typing in WI, brings up windows, TH, brings up thrusters, RE, gets you reactors and refineries, etc by filtering everything else out.

**Comment by Kaiser-Indrasil** [comment]
Yep, additional hotbars and a block search feature would be great quality of life updates. And like you said, you've played for an hour and already needed them.
